data/piyush-multiplexer.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "Piyush Goyani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer | Blogger",
+  "avatar": "https://github.com/piyush-multiplexer.png",
```

The avatar field is no longer required as of a new LinkFree update.
```suggestion
  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer | Blogger"
```

---

Removed. Thanks for letting me know. @kumarsonsoff3 please note!